STX.REST.RESTFulSense.Clients.Tests.Unit/Services/Foundations/HttpExchanges/HttpExchangeServiceTests.Headers.cs

```diff
@@ -0,0 +1,1895 @@
+                })
+                .ToArray();
+
+            ContentDispositionHeader[] invalidContentDispositionHeaders =
+                CreateInvalidContentDispositionHeaders();
+
+            invalidContentDispositionHeaders
+                .Select(invalidContentDispositionHeader =>
+                    {
+                        theoryData.Add(
+                            CreateContentDispositionHeaderException(
+                                invalidContentDispositionHeader));
+
+                        return theoryData;
+                    })
+                    .ToArray();
+
+            string[] invalidContentEncodingHeaders = CreateInvalidStringArrayHeaders();
+            invalidContentEncodingHeaders
+                .Select(invalidContentEncodingHeader =>
+                {
+                    theoryData.Add(
+                        CreateContentEncodingHeaderException(
+                            new string[] { invalidContentEncodingHeader }));
+
+                    return theoryData;
+                })
+                .ToArray();
+
+            string[] invalidContentLanguageHeaders = CreateInvalidStringArrayHeaders();
+            invalidContentLanguageHeaders
+                .Select(invalidContentLanguageHeader =>
+                {
+                    theoryData.Add(
+                        CreateContentLanguageHeaderException(
+                            new string[] { invalidContentLanguageHeader }));
+
+                    return theoryData;
+                })
+                .ToArray();
+
+            ContentRangeHeader[] invalidContentRangeHeaders =
+                CreateInvalidContentRangeHeaders();
+
+            invalidContentRangeHeaders
+                .Select(invalidContentRangeHeader =>
+                {
+                    theoryData.Add(
+                        CreateContentRangeHeaderException(
+                            invalidContentRangeHeader));
+
+                    return theoryData;
+                })
+                    .ToArray();
+
+            MediaTypeHeader[] invalidContentTypeHeaders = CreateInvalidMediaTypes();
+            invalidContentTypeHeaders
+                .Select(invalidContentTypeHeader =>
+                {
+                    theoryData.Add(
+                        CreateContentTypeHeaderException(
+                           invalidContentTypeHeader));
+
+                    return theoryData;
+                })
+                .ToArray();
+
+            return theoryData;
+        }
+
+        private static dynamic CreateSendRequestDependencyExceptions(TheoryData<dynamic> theoryData)
+        {
+            var someMessage = GetRandomString();
+            var httpRequestException = new HttpRequestException();
+
+            var failedRequestHttpExchangeException = new FailedRequestHttpExchangeException(
+                message: "Failed http request error occurred, contact support.",
+                innerException: httpRequestException);
```

Comment:
```suggestion
            var failedRequestHttpExchangeException =
                new FailedRequestHttpExchangeException(
                    message: "Failed http request error occurred, contact support.",
                    innerException: httpRequestException);
```
Do the same for the rest in this method. Consistency across files. See how it was done in the HttpExchangeService.Exceptions file and then here it deviated to an odd break at the open argument parenthesis.